nixos/doc/manual/default.nix

```diff
@@ -36,7 +36,7 @@ let
           package = args.package or (lib.attrByPath path (throw "Invalid package attribute path `${toString path}'") pkgs);
         in "<listitem>"
         + "<para><literal>pkgs.${name} (${package.meta.name})</literal>"
-        + lib.optionalString (!package.meta.evaluates) " <emphasis>[UNAVAILABLE]</emphasis>"
+        + lib.optionalString (!package.meta.available) " <emphasis>[UNAVAILABLE]</emphasis>"
```

Comment:
could the `broken` attribute be re-used there?

Reply:
I think `broken` has a more narrow meaning (this stopped working where it worked before), and the check also includes `platforms` and `license`?